assets/i18n/en_US.json

```diff
@@ -211,7 +211,9 @@
 
         "resetStoredPatchesLabel": "Reset patches",
         "resetStoredPatchesHint": "Reset the stored patches selection",
-        "resetStoredPatchesDialogText": "Are you sure you want to reset the stored patches selection?",
+
+        "resetStoredPatchesDialogTitle": "Reset patches selection?",
+        "resetStoredPatchesDialogText": "Restoring patches selection to default will cause all previously selected patches to be reset.",
```

Comment:
This can simply be `Resetting patches selection will remove all selected patches.`